README.md

```diff
@@ -26,7 +26,8 @@ You can configure `rofi-rbw` either with cli arguments or with a config file cal
 | `--selector-args` |              |                                                      | Define arguments that will be passed through to `rofi` or `wofi`.<br/>Please note that you need to specify it as `--selector-args="<args>"` or `--selector-args " <args>"` because of a [bug in argparse](https://github.com/python/cpython/issues/53580) |
 | `--selector`      |              | `rofi`, `wofi`                                       | Show the selection dialog with this application. Chosen automatically by default.                                                                                                                                                                         |
 | `--clipboarder`   |              | `xsel`, `xclip`, `wl-copy`                           | Access the clipboard with this application. Chosen automatically by default.                                                                                                                                                                              |
-| `--typer`         |              | `xdotool`, `wtype`                                   | Type the characters using this application. Chosen automatically by default.                                                                                                                                                                              |
+| `--typer`         |              | `xdotool`, `wtype`, `ydotool`                        | Type the characters using this application. Chosen automatically by default.                                                                                                                                                                              |
+| `--key-delay`     |              | integer number >= 0 (default is `0`)                 | Define the duration in milliseconds that each key is held for when typing                                                                                                                                                                                 |
```

Comment:
That's not true anymore, is it?

Reply:
Ah yes, I'll correct that now, thank you. The addition of `ydotool` to the possible value of `--typer` is still correct, but not completely relevant to this PR, would you like me to remove that as well and put it in another PR?

Reply:
No, that's fine here, thanks 🙂

Reply:
I've fixed it now :slightly_smiling_face: